docker/Dockerfile

```diff
@@ -60,7 +60,7 @@ ARG INTEL_SCIPY_VERSION="==1.7.3"
 # content of the intel pypi or not. (this Dockerfile currently assumes that
 # it is not limited to intel pypi. The opposite assumption seems to create
 # impossible build conditions.)
-ARG INTEL_NUMBA_VERSION=""
+ARG INTEL_NUMBA_VERSION="<0.57"
```

Comment:
Please link to an issue that explains why it's needed:

- if it's a known issue upstream, just linked to that,
- if it's something we need to updated in our code, maybe open an issue with the traceback one would typically observe if we remove the pinning.

Reply:
See issue OP :grin: